mkdocs.yml

```diff
@@ -383,6 +383,8 @@ pages:
         - Enterprise installations: platform/server/enterprise.md
         - Troubleshooting: platform/server/troubleshooting.md
         - Installer reference: platform/server/install.md
+      - Upgrade instructions:
+        - Upgrading Shippable Server: platform/server/upgrade.md
```

Comment:
I don't think we should make another sub menu for this.... just put Upgrading Shippable Server under installations